### PR TITLE
Document how to reference frontmatter and consts

### PR DESCRIPTION
**Pull request by peduarte**
**What**:

This PR adds a section in the README on how to reference frontmatter meta and consts.

**Why**:

It's a powerful feature that can help people create more maintainable mdx files. We [chatted about it on Twitter](https://twitter.com/kentcdodds/status/1385294146508705797)

**Checklist**:

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged

### CODE REVIEW

**Comment by Arcath** [comment]
Thanks for the PR!

Could you add an example of using the returned values alongside your example mdx?

**Comment by peduarte** [comment]
@Arcath hey 👋 thank you! 

I'd love to, but Im not sure I fully understand. Sorry, would you mind explaining it to me?

**Comment by Arcath** [comment]
Sorry I got  confused I thought you where using them outside the document.

**Comment by peduarte** [comment]
@Arcath Ahh gotcha! Thanks for the explanation 👍

File: README.md [hunk 1]
```diff
@@ -418,6 +419,23 @@ function MDXPage({code}: {code: string}) {
 }
 ```
 
+### Frontmatter and const
+
+You can reference frontmatter meta or consts in the mdx content.
+
+```msx
```
Comment (kentcdodds) [inline README.md:426]:
```suggestion
```mdx
```

**Review by kentcdodds — COMMENTED** [review]
Looks good! Just one small change.

**Comment by kentcdodds** [comment]
@all-contributors please add @peduarte for docs

**Pull request merged**